bucket/wezterm.json

```diff
@@ -0,0 +1,32 @@
+{
+    "version": "20200406-151651-5b700e4",
+    "description": "GPU-accelerated terminal emulator",
+    "homepage": "https://github.com/wez/wezterm",
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/wez/wezterm/releases/download/20200406-151651-5b700e4/WezTerm-windows-20200406-151651-5b700e4.zip",
+            "hash": "5f1b0898c54b0dd69a56c9ee2c2f3d2c4c39f29e2b80acb3320515b833a9f687"
+        }
+    },
+    "extract_dir": "WezTerm-windows-20200406-151651-5b700e4",
+    "bin": "wezterm.exe",
+    "shortcuts": [
+        [
+            "wezterm.exe",
+            "Wezterm"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/wez/wezterm",
+        "regex": "WezTerm-windows-([\\w-]+)"
```

Comment:
I think this was better before; the version format is described here:
https://wezfurlong.org/wezterm/config/lua.html#weztermversion
There are only three components to the version string.

It is important because I have plans to add eg: `.msi` or `.exe` windows downloads in the future and this regex will be broken by that.

Reply:
@wez the man himself, cool 😄 
Would the correspondence between download file name and version still be the same?
The new regex is more lenient, so it should be more flexible in terms of changes to the version format,
as long as the URL to the download can still be string-interpolated in the same way.

I don't mind either way though :wink:

Reply:
I'm not familiar with this configuration in scoop, but my assumption is that the capture defined in this regex is used to populate the `$version` token in the `autoupdate` section below.

If that is the case then it feels best to constrain the regex to match the version triple to avoid problems if/when I start to add files named something like (for a slightly but not overly contrived example): `WezTerm-windows-DATE-TIME-HASH-debuginfo.zip` from falsely matching and being used as a candidate update!